### DISCUSSION TITLE
Suggestion for comment improvement

### DISCUSSION THREAD

**Discussion by fabiocs8** [dbeaver/dbeaver#21562] [Ideas]
When debugging, it is often useful to comment and uncomment slices of the script through` /* code */`. 

In order to only turn comments on and off, I would like to comment   through `/* code/**/`, so that  by only removing  the first `/* ` I would be able to uncomment the hole  code. 

However,  in a situation such as `/*code_01 /**/ code_02`, when I apply the first `/*` (just before code_01), the code parser also comments  the  code_02, instead of  closing the comment section when it first finds `*/` (just before code_02). 

It seems reasonable that code parser would close the comment section as soon as it finds the `*/` marker. 

Thank you.

**Comment by E1izabeth** [comment]
It sounds like a bug. Could you create an issue?
